descriptors/ModuleDescriptor-template.json

```diff
@@ -189,7 +189,7 @@
       ]
     },
     {
-      "id": "organizations-storage.url",
+      "id": "organizations-storage.urls",
```

Comment:
I'm fine with this change, but a little surprised to see this here since it isn't mentioned in the description...

Reply:
PR description updated